### DISCUSSION TITLE
Chat error after changing AI Search Index name

### DISCUSSION THREAD

**Discussion by rayxfelipe** [microsoft/PubSec-Info-Assistant#690] [Q&A]
I changed the index name in the web app (both of them) and afterwards, I get an error on the chat when posting questions: "Error: Unknown Error".

If I change the index name, is there anywhere else I need to make this change? The index does exist, in addition to the default index, i.e. vector-index.

Thanks.

**Comment by dayland** [comment]
You cannot just use any search index. The fields on the index need to be in a specific name and format to be compatible with Info Assistant. The name of certain index fields are in app service environment variables and are used throughout the web backend and frontend code.

**Reply by rayxfelipe** [reply]
Follow up question. Does it have to have the same number of fields? I need to get it to work with another application requiring an index, but that application has a different Index Field requirements. No updates on the existing Pubsec IA index, but I plan to add fields for another app. Will this work? Thanks.
